Header  header
int32   gpsweek
float64 gpstime
float32 heading
float32 pitch
float32 roll
float32 wx
float32 wy
float32 wz
float32 ax
float32 ay
float32 az
float64 lat
float64 lon
float64 alt
float32 ve
float32 vn
float32 vu
float32 v
int16   nsv1
int16   nsv2
uint8   status
int16   age
int16   warning
